packages/ckeditor5-block-quote/src/blockquoteediting.js

```diff
@@ -78,17 +78,19 @@ export default class BlockQuoteEditing extends Plugin {
 						const range = writer.createRangeIn( element );
 
 						for ( const child of range.getItems() ) {
-							if ( child.is( 'blockQuote' ) && !schema.checkChild( writer.createPositionBefore( child ), child ) ) {
-								writer.unwrap( child );
+							if ( child.is( 'element', 'blockQuote' ) ) {
```

Comment:
We could have something like this:

```javascript
if (
	child.is( 'element', 'blockQuote' ) &&
	!schema.checkChild( writer.createPositionBefore( child ), child )
) {
	writer.unwrap( child );

	return true;
}
```

since I don't like double `if`s - but again this is trival and I'd pass that as-is.